### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**

### CODE REVIEW

File: scripts/_common.sh [hunk 1]
```diff
@@ -1,31 +1,15 @@
 #!/bin/bash
 
 #=================================================
-# PERSONAL HELPERS
+# COMMON VARIABLES
 #=================================================
 
+app_depencencies="default-jre-headless daemon net-tools"
```
Comment (kay0u) [inline scripts/_common.sh:7]:
```suggestion
app_depencencies="openjdk-11-jdk daemon net-tools"
```

I wanted to apply #31 but the corresponding PR have some trouble, so I just applying the commit

**Comment by kay0u** [comment]
!testme

File: scripts/remove [hunk 2]
```diff
@@ -24,10 +24,10 @@ domain=$(ynh_app_setting_get --app=$app --key=domain)
 # REMOVE SERVICE FROM ADMIN PANEL
 #=================================================
 
-# Check if the service is declared in YunoHost
-if yunohost service status $app >/dev/null 2>&1
+# Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
+if ynh_exec_warn_less yunohost service status $app >/dev/null
 then
-	ynh_script_progression --message="Removing $app service..."
+	ynh_script_progression --message="Removing $app service integration..." --time --weight=1
```
Comment (ericgaspar) [inline scripts/remove:30]:
```suggestion
	ynh_script_progression --message="Removing $app service integration..." --weight=1
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

**Comment by yalh76** [comment]
!testme

File: scripts/install [hunk 3]
```diff
@@ -150,7 +148,7 @@ yunohost app ssowatconf
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
 
-yunohost service add $app --log /var/log/$app/$app.log
+yunohost service add $app --log=/var/log/$app/$app.log
```
Comment (yalh76) [inline scripts/install:151]:
```suggestion
yunohost service add $app --log /var/log/$app/$app.log
```
Not sure if `=` is required

File: scripts/restore [hunk 4]
```diff
@@ -123,7 +118,7 @@ ynh_restore_file --origin_path=/etc/default/$app
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
 
-yunohost service add $app --log "/var/log/$app/$app.log"
+yunohost service add $app --log="/var/log/$app/$app.log"
```
Comment (yalh76) [inline scripts/restore:121]:
```suggestion
yunohost service add $app --log "/var/log/$app/$app.log"
```

File: scripts/upgrade [hunk 5]
```diff
@@ -190,6 +188,12 @@ done
 mv /var/lib/jenkins/config.xml.bak /var/lib/jenkins/config.xml
 ynh_systemd_action --action=restart --line_match="Jenkins is fully up and running" --log_path="/var/log/$app/$app.log" --timeout="3600"
 
+#=================================================
+# ADVERTISE SERVICE IN ADMIN PANEL
+#=================================================
+
+yunohost service add $app --log=/var/log/$app/$app.log
```
Comment (yalh76) [inline scripts/upgrade:195]:
```suggestion
yunohost service add $app --log /var/log/$app/$app.log
```

**Review by yalh76 — CHANGES_REQUESTED** [review]
Very Quick

**Review by yalh76 — APPROVED** [review]

**Review by yalh76 — APPROVED** [review]

**Comment by ericgaspar** [comment]
The "old" CI don't fail on upgrade...

**Pull request merged**